### PR TITLE
Add Customize DropDown menu article to Getting Started guides

### PR DESCRIPTION
**Pull request by czernous-devexpress**

### CODE REVIEW

File: includes/common-customize-dropdown-menu.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+On desktops and iOS devices, the drop-down menu is the [Popover](/concepts/05%20Widgets/Popover/00%20Overview.md '/Documentation/Guide/UI_Components/Popover/Overview/') UI component; on other devices, it is the [Popup](/concepts/05%20Widgets/Popup/00%20Overview.md '/Documentation/Guide/UI_Components/Popup/Overview/') UI component. To use the Popup on all devices, assign **false** to the [usePopover](/api-reference/10%20UI%20Widgets/dxLookup/1%20Configuration/usePopover.md '/Documentation/ApiReference/UI_Components/dxLookup/Configuration/#usePopover') property.
+
+To customize the Popup or Popover, use the [dropDownOptions](/api-reference/10%20UI%20Widgets/dxLookup/1%20Configuration/dropDownOptions.md '/Documentation/ApiReference/UI_Components/dxLookup/Configuration/#dropDownOptions') object.
```
Comment (RomanTsukanov) [inline includes/common-customize-dropdown-menu.md]: You need links with {WidgetName}.

File: includes/common-customize-dropdown-menu.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+On desktops and iOS devices, the drop-down menu is the [Popover](/concepts/05%20Widgets/Popover/00%20Overview.md '/Documentation/Guide/UI_Components/Popover/Overview/') UI component; on other devices, it is the [Popup](/concepts/05%20Widgets/Popup/00%20Overview.md '/Documentation/Guide/UI_Components/Popup/Overview/') UI component. To use the Popup on all devices, assign **false** to the [usePopover](/api-reference/10%20UI%20Widgets/dx{WidgetName}/1%20Configuration/usePopover.md '/Documentation/ApiReference/UI_Components/dx{WidgetName}p/Configuration/#usePopover') property.
```
Comment (RomanTsukanov) [inline includes/common-customize-dropdown-menu.md]:
```suggestion
On desktops and iOS devices, the drop-down menu is the [Popover](/concepts/05%20Widgets/Popover/00%20Overview.md '/Documentation/Guide/UI_Components/Popover/Overview/') UI component; on other devices, it is the [Popup](/concepts/05%20Widgets/Popup/00%20Overview.md '/Documentation/Guide/UI_Components/Popup/Overview/') UI component. To use the Popup on all devices, assign **false** to the [usePopover](/api-reference/10%20UI%20Widgets/dx{WidgetName}/1%20Configuration/usePopover.md '/Documentation/ApiReference/UI_Components/dx{WidgetName}/Configuration/#usePopover') property.
```

File: includes/common-customize-dropdown-menu.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+On desktops and iOS devices, the drop-down menu is the [Popover](/concepts/05%20Widgets/Popover/00%20Overview.md '/Documentation/Guide/UI_Components/Popover/Overview/') UI component; on other devices, it is the [Popup](/concepts/05%20Widgets/Popup/00%20Overview.md '/Documentation/Guide/UI_Components/Popup/Overview/') UI component. To use the Popup on all devices, assign **false** to the [usePopover](/api-reference/10%20UI%20Widgets/dx{WidgetName}/1%20Configuration/usePopover.md '/Documentation/ApiReference/UI_Components/dx{WidgetName}/Configuration/#usePopover') property.
```
Comment (flagmanAndrew) [inline includes/common-customize-dropdown-menu.md]: Afaik, only Lookup editor uses the Popover component by default. Other drop-down editors use Popup unless usePopover is specified

File: includes/common-customize-dropdown-menu.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+The drop-down menu is the [Popup](/concepts/05%20Widgets/Popup/00%20Overview.md '/Documentation/Guide/UI_Components/Popup/Overview/') UI component.
+
+To customize the Popup, use the [dropDownOptions](/api-reference/10%20UI%20Widgets/dx{WidgetName}/1%20Configuration/dropDownOptions.md '/Documentation/ApiReference/UI_Components/dx{WidgetName}/Configuration/#dropDownOptions') object.
```
Comment (RomanTsukanov) [inline includes/common-customize-dropdown-menu.md]:
```suggestion
The {WidgetName} uses the [Popup](/concepts/05%20Widgets/Popup/00%20Overview.md '/Documentation/Guide/UI_Components/Popup/Overview/') component as the drop-down menu. To customize the menu, specify [Popup properties](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/) in the [dropDownOptions](/api-reference/10%20UI%20Widgets/dx{WidgetName}/1%20Configuration/dropDownOptions.md '/Documentation/ApiReference/UI_Components/dx{WidgetName}/Configuration/#dropDownOptions') object.
```

**Review by RomanTsukanov — APPROVED** [review]

**Pull request merged**